images/cpg-pipes/Dockerfile

```diff
@@ -1,13 +1,18 @@
 # Add cpg_pipes to driver-base
 FROM australia-southeast1-docker.pkg.dev/analysis-runner/images/driver-base:1.1
-# hadolint ignore=DL3008, DL3013
+
+ENV TZ=Australia \
+    DEBIAN_FRONTEND=noninteractive
+# hadolint ignore=DL3008, DL3013, DL4006
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    software-properties-common \
     gcc \
     build-essential \
-    python3 \
-    python3-pip \
-    python3-dev \
+    curl \
     && \
-    pip install --no-cache-dir cpg-pipes==0.3.1 && \
+    add-apt-repository ppa:deadsnakes/ppa -y && \
+    apt-get install -y --no-install-recommends python3.10 python3.10-distutils python3-apt python3.10-dev  && \
+    curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10 && \
```

Comment:
Can't we install pip for python 3.10 with apt-get?

Reply:
E.g. even if `python3.10` is installed, `python3-pip` would still somehow use python 3.8?